Added reader info support for UHFReader86

DylanDonkersgoed opened the pull request.

wabson commented:
Thanks for the PR! I've not tested before with the CF-MU904 but looks like a nice bit of hardware so have ordered one to try out, along with the interface board.

In the meantime it would be great to add these changes to the library, just one thing I'm not sure of, could you explain what the `uhfreader18.py` file is for?

DylanDonkersgoed commented:
Don't mind that, still working on this. I copied the wrong file to the wrong place. I've added the proper uhfreader86.py file but it doesn't do too much at the moment. I added it as a stub to extend as necessary. Plus it was bugging me that I had to import the uhfreader288m stuff for a non-uhfreader288m reader.

I'm planning to build out my app a little more and expand the PR as needed. It will be doing more than just the inventory command - probably stop it beeping at least, maybe adjust some other configuration, maybe write some data (haven't decided whether I want to write data to the tags or keep a separate database of what corresponds to the IDs).
